contrib/signet/README.md

```diff
@@ -81,3 +81,4 @@ These steps can instead be done explicitly:
 
 This is intended to allow you to replace part of the pipeline for further experimentation (eg, to sign the block with a hardware wallet).
 
+For custom signets with a trivial challenge such as  `OP_TRUE` the walletprocesspsbt step can be skipped.
```

Comment:
I would rephrase as "the walletprocesspsbt step *has* to be skipped", as it currently returns an error:

```
❯ $CLI -signet getblocktemplate '{"rules": ["signet","segwit"]}' | $MINER --cli="$CLI" genpsbt --address="$ADDR" | $CLI -signet -stdin walletprocesspsbt
error code: -22
error message:
Specified sighash value does not match value stored in PSBT	
```

Reply:
> sighash value does not match

That's a bug I think, but I'm not sure what's causing it.

Reply:
I think walletprocesspsbt is trying to sign the PSBT using `SIGHASH_DEFAULT`, while the PSBT specifies to use `SIGHASH_ALL`.

Using `$CLI -signet walletprocesspsbt $PSBT true ALL` works for me.